ni_measurementlink_service/_annotations.py

```diff
@@ -0,0 +1,5 @@
+# Constants for annotations
```

Comment:
Change the comment into a docstring.
```suggestion
"""Constants for annotations."""
```